packages/sdk/src/__tests__/getPackedPackage.test.ts

```diff
@@ -4,6 +4,8 @@ import fs from 'fs'
 
 describe('getPackedPackage', () => {
   it('test argument vulnerability', async () => {
+    jest.setTimeout(10000)
```

Comment:
Can `jest.setTimeout` be invoked inside a test? I thought it's only supposed to be called globally before the tests, and for individual tests there's the third argument to `it`/`test`, like this:

```ts
it('test argument vulnerability', async () => { }, 10000)
```

(Not saying `jest.setTimeout(10000)` won't work inside the test though, I never tried doing this)

Reply:
We use it in both cases and it works the same, though I also like the third argument we don't have a guideline right now which one we should use and if we should use the same one everywhere.